t/performance_probability.t

```diff
@@ -0,0 +1,89 @@
+#Performance Probability test.
+use 5.010;
+use strict;
+use warnings;
+
+use Date::Utility;
+
+use Test::Most;
+use Test::FailWarnings;
+use Test::More;
+
+BEGIN {
+# Temporary while testing if perl-Math-BivariateCDF can be build on travis.
```

Comment:
No need of that. `M::BivariateCDF` should be pulled in as as regular CPAN-dependency. I.e. `M::BivariateCDF` should be released first.